src/main/java/net/glowstone/command/minecraft/DefaultGameModeCommand.java

```diff
@@ -0,0 +1,75 @@
+package net.glowstone.command.minecraft;
+
+import org.bukkit.Bukkit;
+import org.bukkit.ChatColor;
+import org.bukkit.GameMode;
+import org.bukkit.command.CommandSender;
+import org.bukkit.command.defaults.VanillaCommand;
+import org.bukkit.util.StringUtil;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+public class DefaultGameModeCommand extends VanillaCommand {
+    private static final List<String> GAMEMODES = Arrays.asList("adventure", "creative", "survival", "spectator");
```

Comment:
Declare this array in CommandUtils, and use it for both `/gamemode` and `/defaultcommand`. Also, make it an unmodifiable list.